Mathlib/Analysis/SpecialFunctions/ExpDeriv.lean

```diff
@@ -174,6 +177,14 @@ theorem ContDiffWithinAt.cexp {n} (hf : ContDiffWithinAt 𝕜 n f s x) :
 
 end
 
+open Complex in
+theorem iteratedDeriv_cexp_const_mul (n : ℕ) (c : ℂ) :
```

Comment:
I think the only real reason to have it (separate from the more general one) is if we mark it `simp`.
```suggestion
@[simp]
theorem iteratedDeriv_cexp_const_mul (n : ℕ) (c : ℂ) :
```